### ISSUE TITLE
Fresh installation to WAMP

### ISSUE THREAD

**Issue by vovpff** [YetiForceCompany/YetiForceCRM#1032]
Server: win7, apache 2.4, php 5.6.9, mysql

I don't have idea how to avoid this at configeration check step before installation:
cURL support True No - in my php.ini extension activated
display_errors	Off On - in my php.ini this parameter turned off, but installation recognize it like on

Frash installation takes some time (5-10 minutes). 

After installation I found some dismissed functions:
1) No possibility to add widgets to "My home page"
2) No possibility to add or change filters for listview of module.
3) Then I trying to add new record to some modules there is appears error message like this:
{"success":false,"error":{"code":"Record you are trying to access is not found: 109 Products","message":"Record you are trying to access is not found: 109 Products"}}
It depends from filled fields. For examle I don't need to use pcice information in my system and newer use price fields in product module. In this build of yetiforce price field not mandatory but with them not possible to add record to module.
4) There is not possible to add Project task. I filled all possible fields but everytime the same error like in line 3.

**Comment by waw555** [comment]
1. https://test.yetiforce.com/index.php?module=WidgetsManagement&parent=Settings&view=Configuration&block=2&fieldid=67

Add widget for role

**Comment by vovpff** [comment]
Thanks a lot. It works! What about another not working features?

**Comment by waw555** [comment]
2. https://test.yetiforce.com/index.php?module=CustomView&parent=Settings&view=Index&block=2&fieldid=57

**Comment by waw555** [comment]
add in site file phpinfo.php
add in file phpinfo.php
<?php
phpinfo();
?>
see parametr 
cURL
display_errors

**Comment by KatarzynaUlichnowska** [comment]
Hello,
which version do you use? Please verify if the problems also exist on the latest v 2.0.75.

**Comment by vovpff** [comment]
display error:

![image](https://cloud.githubusercontent.com/assets/8545791/8052320/014fa580-0e8f-11e5-96a9-644d2d4072ba.png)

cURL has only one mention

![image](https://cloud.githubusercontent.com/assets/8545791/8052344/44b4b40a-0e8f-11e5-9dc3-2ebf05aa9f68.png)

**Comment by vovpff** [comment]
How to get latest version? Download master.zip?

**Comment by mariuszkrzaczkowski** [comment]
Yes, https://github.com/YetiForceCompany/YetiForceCRM/archive/master.zip

**Comment by vovpff** [comment]
I try it. Looks like there in no difference for next points:

 2) No possibility to add or change filters for listview of module.
3) Then I trying to add new record to some modules there is appears error message like this:
{"success":false,"error":{"code":"Record you are trying to access is not found: 109 Products","message":"Record you are trying to access is not found: 109 Products"}}
It depends from filled fields. For examle I don't need to use pcice information in my system and newer use price fields in product module. In this build of yetiforce price field not mandatory but with them not possible to add record to module.
4) There is not possible to add Project task. I filled all possible fields but everytime the same error like in line 3.

**Comment by bpabiszczak** [comment]
Problem with cURL: http://stackoverflow.com/questions/13021536/how-to-enable-curl-in-wamp-server

**Comment by vovpff** [comment]
Another one WAMP installation:

All requested parameters is Ok. cURL too.  Looks like installation stops without any message. Login page is working but data in db for users is missing. Whats wrong?

**Comment by Armaia** [comment]
Hi vovpff,
Did you solve your problem? 
I have exactly the same problem and I have working the vtiger crm on the same server without problems.

**Comment by vovpff** [comment]
Hi Armaia,
For this time no. This time I don't have a lot of time for this. But i will try to solve it maybe next week.

**Comment by mariuszkrzaczkowski** [comment]
Does the problem still exist? If it does, download the latest version and send us error logs.

**Comment by Armaia** [comment]
Hi Mariusz,

 

Yes. The problema still exists. I tried again yesterday with the last version. I opened an issue: #1139.

Best Regards,

 

Arlindo Maia

 

De: Mariusz Krzaczkowski [mailto:notifications@github.com] 
Enviada: 24 de junho de 2015 08:51
Para: YetiForceCompany/YetiForceCRM
Cc: Armaia
Assunto: Re: [YetiForceCRM] Fresh installation to WAMP (#1032)

 

Does the problem still exist? If it does, download the latest version and send us error logs.
